src/Pages/OmVirksomheten/Kontaktinfo.tsx

```diff
@@ -116,20 +118,28 @@ export const KontaktinfoUnderenhet = () => {
     if (kontaktinfo === null) return null;
     if (kontaktinfo.eposter.length === 0 && kontaktinfo.telefonnumre.length === 0) return null;
     return (
-        <div className="kontaktinfo">
+        <Tekstboks className="kontaktinfo">
             <TittelMedHjelpetekst>Varslingsadresser for underenhet</TittelMedHjelpetekst>
             <KontaktinfoListe kontaktinfo={kontaktinfo} />
             <AltinnLenke />
-        </div>
+        </Tekstboks>
     );
 };
 
-export const KontaktinfoHovedenhet = () => {
+export const KontaktinfoOverordnetEnhet = ({
+    overordnetEnhet,
+}: {
+    overordnetEnhet: Hovedenhet;
+}) => {
     const kontaktinfo = useKontaktinfo()?.hovedenhet ?? null;
     if (kontaktinfo === null) return null;
+    const orgType =
+        overordnetEnhet.organisasjonsform?.kode === 'ORGL' ? 'Organisasjonsledd' : 'Hovedenhet';
```

Comment:
Skrive med liten `O` og `H`, så slipper vi `toLowerCase()`?

Reply:
Det gikk nok litt fort her. Brukes med stor O og H i OverordnetEnhet, men ikke i kontaktinfo! 👍

Reply:
Samme gjelder nok også for Kontonummer